markdown/usage/troubleshooting.md

```diff
@@ -24,6 +28,138 @@ subtitle: How to troubleshoot common mistakes and issues
 - [Using a local version of iGenomes](#using-a-local-version-of-igenomes)
 - [Extra resources and getting help](#extra-resources-and-getting-help)
 
+
+## Troubleshooting basics
+
+These are the recommended steps for troubleshooting a pipeline. 
+
+### Start small
+
+Before using the pipeline with your data, make sure to run a test using: 
+
+```bash
+nextflow run nf-core/<pipeline_name> -profile test,docker
+```
+
+If Docker is not installed, you can replace `docker` with `singularity` or `conda`, see
+the [Getting Started](https://nf-co.re/usage/introduction) tutorial for further information. If a test fails, it might indicate that 
```

Comment:
```suggestion
the [Getting Started](https://nf-co.re/usage/introduction) tutorial for further information. If a test fails, it might indicate that
```